app/views/admin/users/export.csv.erb

```diff
@@ -0,0 +1,6 @@
+<%- headers = ["Name", "Username", "Email address", "Status", "Joining date", "Last activity", "Organizations"] -%>
+<%= CSV.generate_line headers %>
+<%- @users.each do |user| -%>
+  <%- row = [user.name, user.username, user.email, user_status(user) , user.registered_at.strftime("%d %b, %Y"), user.last_activity.strftime("%d %b, %Y"), user.organizations.pluck(:name)] -%>
```

Comment:
`strftime("%d %b, %Y")` can also be extracted into a helper as it's been used in the member view in multiple places. I've made a note of it and I'll create a subsequent PR to abstract it out.